README.md

```diff
@@ -30,54 +30,54 @@
 
 ### ❓ Why
 
-No matters your religion, IMAP4 or HTTP, you should not worries about accessing easily header and associated attributes, adjectives or values.
+No matter your religion or choice of IMAP4 or HTTP, you should not worry about easily accessing header and associated attributes, adjectives or values.
```

Comment:
I think I was misunderstood here ^^. I wanted to compare IMAP4 and HTTP as a religion. *some kind of joke*. 
What I actually wanted to say is : *No matter if you are currently building software using HTTP or IMAP communications."

Reply:
Ah, sorry for misunderstanding. Changed to what you meant to say.